pynhm/utils/dis_hru.py

```diff
@@ -0,0 +1,212 @@
+import numpy as np
```

Comment:
this is what remains from me "rolling my own", this will tag along until we have an HRU based package to test